local/gugcat/templates/gcat_edit_form.mustache

```diff
@@ -0,0 +1,54 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
```

Comment:
As much as possible, please reuse the add_form.mustache, I suggest rename it to addedit_form to make it generic for both adding and updating grades.